docs/accelerator/README.md

```diff
@@ -0,0 +1,66 @@
+# Landing Zone Accelerator
+
+## Background
+
+Collie and its Collie Hub, with all of its available kit modules, are open-source and fully usable for free.
+Anyone can use Collie and the Collie Hub to build and manage their cloud foundation and their landing zones.
+
+Collie and the Collie Hub are offered by [meshcloud](https://meshcloud.io/en/), which is a technology startup
+that offers a Cloud Foundation Platform as a Software-as-a-Service.
+
+To get organizations up-to-speed even quicker, meshcloud offers the
+[Azure Landing Zone Accelerator](https://www.meshcloud.io/en/services/landing-zone-accelerator/) as a service to get
+an up-and-running Azure Cloud Foundation Architecture within 6 weeks.
+
+## The Azure Landing Zone Accelerator
+
+The Azure Landing Zone Accelerator is **the** way to get your organization to Azure, fast and efficiently.
+By doing a number of hands-on workshops with the cloud experts from meshcloud, and the powerful
+Collie & Collie Hub technology, organizations can get their Azure Cloud Foundation built incredibly fast.
+
+### Why the Azure Landing Zone Accelerator?
+
+- **Based on open-source IaC tools**: The entire Accelerator uses open-source technology like
+  Collie, and industry-leading IaC tool Terraform, which prevents you from being locked-in to tools.
```

Comment:
I dont get it, because you dependent to terraform. It allows you to doing IaC over multiple vendors. I would 
 say:  "you are not depent on the tools of the vendors (like cloudformation, biceps and so on".